.s2i/bin/run

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+
+set -e
+
+if [[ "$DATABASE_HOST" =~ '_SERVICE_HOST' ]]; then
+    DATABASE_HOST=$(echo "$DATABASE_HOST" | tr '[:lower:]' '[:upper:]')
+    export DATABASE_HOST=${!DATABASE_HOST}
+    echo "---> DATABASE_HOST: ${DATABASE_HOST} ..."
+fi
+
+if [[ "$DATABASE_PORT" =~ '_SERVICE_PORT' ]]; then
+    DATABASE_PORT=$(echo "$DATABASE_PORT" | tr '[:lower:]' '[:upper:]')
+    export DATABASE_PORT=${!DATABASE_PORT}
+    echo "---> DATABASE_PORT: ${DATABASE_PORT} ..."
+fi
+
+if [ -z "${DO_NOT_CREATE_DATABASE}" ]; then
+    echo "---> Creating database ..."
+    bundle exec rake db:create
+fi
+
+if [ -z "${DO_NOT_MIGRATE}" ]; then
+    echo "---> Migrating database ..."
+    bundle exec rake db:migrate
+fi
+
+if [ -z "${DO_NOT_SEED}" ]; then
+    echo "---> Seeding database ..."
+    set +e
+    bundle exec rake db:seed
+    set -e
+fi
+
+# Configure the unicorn server
+mv config/unicorn.rb.example config/unicorn.rb
```

Comment:
Sorry I overlooked those lines, `unicorn.rb` isn't used so I think we can remove this 4 files.

Reply:
Since now user can specify whether a pod is web or worker via `WORKER_CMD`, they can do `unicorn -c config/unicorn.rb --listen 0.0.0.0:8080` if this file is ready to serve.

Reply:
I see that makes sense.